### DISCUSSION TITLE
PostgreSQL Data Dump

### DISCUSSION THREAD

**Discussion by mferro64** [openstates/issues#744] [General] [answered]
How often is the PostgreSQL Data Dump updated? We are looking at different integration options - direct API as well as the data dump.

Thanks,

Mike

**Comment by NewAgeAirbender** [comment, marked as answer]
Hey Mike, the PostgreSQL Data is updated monthly, so you can just change the date on this link to whatever time you need https://data.openstates.org/postgres/monthly/2022-06-public.pgdump

**Reply by mferro64** [reply]
Thanks!
